python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -43,19 +43,31 @@ def __init__(self, algo = 'csvv2', verbose = 0):
 
         # define measurement type for each flavor
         self.inputFilePath = os.environ['CMSSW_BASE'] + "/src/PhysicsTools/NanoAODTools/data/btagSF/"
-        self.inputFileName = None
+        self.inputFileName = sfFileName
         self.measurement_types = None
         if self.algo == "csvv2":
-            self.inputFileName = "btagSF_CSVv2_ichep2016.csv"
+            if self.inputFileName is None:
+                self.inputFileName = "CSVv2_94XSF_V1_B_F.csv"
             print("Loading btagSF weights for CSV (v2) algorithm from file '%s'" % os.path.join(self.inputFilePath, self.inputFileName))
             self.measurement_types = {
                 0 : "comb",  # b
                 1 : "comb",  # c
                 2 : "incl"   # light
             }
+        elif self.algo == "deepcsv":
+            if self.inputFileName is None:
+                self.inputFileName = "DeepCSV_94XSF_V1_B_F.csv"
+            print("Loading btagSF weights for deep-CSV (b) algorithm from file '%s'" % os.path.join(self.inputFilePath, self.inputFileName))
+            self.measurement_types = {
+                0 : "comb",  # b
+                1 : "comb",  # c
+                2 : "incl"   # light
+            }
         elif self.algo == "cmva":
-            self.inputFileName = "btagSF_cMVAv2_ichep2016.csv"
```

Comment:
this is changing the default behaviour.

Can you please make it such 

-btagSF() still creates 2016 SF while (or remove the btagSF() and create a btagSF2016() )
-btagSF2017() has default config for 2017.

Reply:
I added btagSF2016() and btagSF2017() and removed btagSF()